frontend/src/views/sql-editor/SQLEditorPage.vue

```diff
@@ -27,13 +34,16 @@
 
 <script lang="ts" setup>
 import { computed } from "vue";
+import { Splitpanes, Pane } from "splitpanes";
+import "splitpanes/dist/splitpanes.css";
```

Comment:
When import this in some other component file, it will cause unwanted style missing. 

And I find use `@import "splitpanes/dist/splitpanes.css";` in style block will resolve this.

Reply:
Thanks. Updatd.